core/store/migrate/migrations/0215_functions_subscriptions.sql

```diff
@@ -0,0 +1,18 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE functions_subscriptions(
+    subscription_id bigint PRIMARY KEY,
```

Comment:
subscription IDs are no longer unique since we have router contract address right?